client/my-sites/stats/stats-chart-tabs/utility.js

```diff
@@ -42,8 +42,35 @@ export function getQueryDate( queryDate, timezoneOffset, period, quantity ) {
 	return endOfPeriodDate;
 }
 
+/**
+ * Wraps a function in a utility method that remembers the last invocation's
+ * arguments and results, and returns the latter if the former match.
+ *
+ * @param {Function} fn The function to be wrapped.
+ *
+ * @returns {Function} The wrapped function.
+ */
+function memoizeLast( fn ) {
```

Comment:
Should I extract this as a lib?

Reply:
Yes, `lib/memoize-last` or `lib/memoize-last-args` would be nice ;)

Reply:
Done! Added some tests and a `README.md`, too :)